modules/nf-core/slamdunk/map/main.nf

```diff
@@ -0,0 +1,51 @@
+process SLAMDUNK_MAP {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/slamdunk:0.4.3--py_0':
+        'biocontainers/slamdunk:0.4.3--py_0' }"
+
+    input:
+    tuple val(meta), path(input)
+    tuple val(meta2), path(fasta)
+
+    output:
+    tuple val(meta), path("outputs/*.bam"), emit: bam
+    path "versions.yml"                   , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    slamdunk \\
+        map \\
+        -r $fasta \\
```

Comment:
Technically this should be `${fasta}` if we follow Nextflow's preferred syntax - do you use VScode as your IDE? If yes you can install the Nextflow extension and set up autoformatting for you :)